ci-ignore-list.yaml

```diff
@@ -0,0 +1 @@
+image-builder-frontend: 'Red Hat Konflux / image-builder-bonfire-tekton / image-builder-frontend'
```

Comment:
I think that's good for now, but can we read the required checks from the API to avoid double-configuration?

somewhere here?
https://docs.github.com/en/rest/branches/branch-protection?apiVersion=2022-11-28#get-status-checks-protection

Reply:
Yup, let's consider that as a follow-up. I'll create an issue.

Reply:
See !39